backend/bamboo/blueprints/schedule_item.py

```diff
@@ -0,0 +1,43 @@
+from apiflask import APIBlueprint
+
+from bamboo.database import db
+from bamboo.database.models import ScheduleItem
+from bamboo.schemas.schedule_item import ScheduleItemIn, ScheduleItemOut
+
+schedule_item = APIBlueprint("schedule_item", __name__)
+
+
+@schedule_item.get("/<int:schedule_item_id>")
+@schedule_item.output(ScheduleItemOut)
+def get_schedule_item(schedule_item_id):
+    return ScheduleItem.query.get_or_404(schedule_item_id)
```

Comment:
```suggestion
    return db.get_or_404(ScheduleItem, schedule_item_id)
```

Please use the SQLAlchemy 2.0 style API: https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#migration-orm-usage